zqd/api/api.go

```diff
@@ -151,6 +156,7 @@ type PcapPostStatus struct {
 	PcapReadSize  int64      `json:"pcap_read_size" unit:"bytes"`
 	SnapshotCount int        `json:"snapshot_count"`
 	Span          *nano.Span `json:"span,omitempty"`
+	Warning       string     `json:"warning,omitempty"`
```

Comment:
I do not believe this is being used, nor do I believe this should go in `PcapPostStatus`.

Reply:
Oops, changed the approach and missed that, thanks